packages/amplify-graphql-auth-transformer/src/vtl-generator/rds/resolvers/query.ts

```diff
@@ -0,0 +1,28 @@
+import { compoundExpression, methodCall, printBlock, ref, set } from 'graphql-mapping-template';
+import { TransformerContextProvider } from '@aws-amplify/graphql-transformer-interfaces';
+import { FieldDefinitionNode, ObjectTypeDefinitionNode } from 'graphql';
+import { ConfiguredAuthProviders, RoleDefinition } from '../../../utils';
+import { constructAuthFilter, generateAuthRulesFromRoles, validateAuthResult } from './common';
+
+export const generateAuthExpressionForQueries = (
+  ctx: TransformerContextProvider,
+  providers: ConfiguredAuthProviders,
+  roles: Array<RoleDefinition>,
+  fields: ReadonlyArray<FieldDefinitionNode>,
+  def: ObjectTypeDefinitionNode,
+  indexName: string | undefined,
+): string => {
+  const expressions = [];
```

Comment:
Most of these inputs are not being used in the method, can you remove unused ones?

Reply:
DDB and RDS VTL generator implement a common interface, most of these arguments are used in DDB but not in RDS. So, there is going to be unused arguments somewhere. We can avoid this by creating datasource specific config types (may be in a separate PR)